services/provider/proto/provider.proto

```diff
@@ -159,6 +163,8 @@ message ReportStatusRequest{
     string storageConsumerUUID = 1;
     string clientPlatformVersion = 2;
     string clientOperatorVersion = 3;
+    string clusterId = 4;
+    string storageClientNamespacedName = 5;
```

Comment:
I don't see a need to report these as part of the status request (they are constant and static)